CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+## Contributing
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+We accept pull requests for bug fixes and small features where the approach will be simple and straightforward. We'd also love to hear about ideas for new features as issues.
+
+We track issues on our project board [here](https://github.com/OffchainLabs/arbitrum-token-bridge/issues).
+
+Please do:
+
+- Check existing issues to verify that the bug or feature request has not already been submitted.
+- Open an issue if things aren't working as expected.
+- Open an issue to propose a significant change.
+- Open a pull request to fix a bug.
+- If you're contributing for the first time here, try and look for any issues labelled [good first issue](https://github.com/OffchainLabs/arbitrum-token-bridge/labels/good%20first%20issue).
```

Comment:
would be nice to link to this page instead
https://github.com/OffchainLabs/arbitrum-token-bridge/contribute

Reply:
Nice. Done!